crates/sdk/src/keygen/mod.rs

```diff
@@ -155,6 +156,77 @@ where
     }
 }
 
+pub fn check_recursive_verifier_size<SC: StarkGenericConfig>(
+    vk: &MultiStarkVerifyingKey<SC>,
+    fri_params: FriParameters,
+    next_log_blowup: usize,
```

Comment:
what does next refer to here?

Reply:
This is the `log_blowup` of the verifier of the output. It determines the maximum trace height that, say, `FriOpeningChip` can have.

Reply:
makes sense, leave a comment